app/controllers/home_controller.rb

```diff
@@ -18,9 +18,11 @@ def sell; end
 
   private
 
-  # Cache the value of the query count for 24 hours
+  # Cache the value of the query count
   def cached_count(key, query)
-    Rails.cache.fetch("home_stats_count_#{key}", expires_in: 1.day, race_condition_ttl: 10) do
+    CacheService.cache("home_stats_count_#{key}",
+                       expires_in: CacheService::HOME_STATS_EXPIRY,
+                       race_condition_ttl: 10) do
```

Comment:
I think the service boundary here isnt good.
If CacheService knows about HOME_STATS_EXPIRY then it can know about it all. I'd do:
CacheService.home_stats_count do
    ...

otherwise I'd move HOME_STATS_EXPIRY to the home controller.

Reply:
agree. I pointed something similar in the original PR.